src/test/java/com/codurance/retropolis/acceptance/board/BoardStepDefsIntegrationTest.java

```diff
@@ -52,4 +58,46 @@ public void theClientReceivesBoardWithThreeColumnsAnd(String firstTitle, String
             new Column(3L, thirdTitle, Collections.emptyList()))))));
   }
 
+  @Given("a user has accessed the test board")
+  public void aUserHasAccessedTheTestBoard() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, TOKEN);
+    HttpWrapper.executeGet(url + "/boards/1", headers);
+  }
+
+  @And("a user has previously accessed the board")
+  public void aUserHasPreviouslyAccessedTheBoard() {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, TOKEN);
+    HttpWrapper.executeGet(url + "/boards/1", headers);
+  }
+
+  @When("the user requests all their boards")
+  public void theUserRequestsAllTheirBoards() {
+    HttpHeaders headers = new HttpHeaders();
```

Comment:
Can you please move those headers to the `Before` function? You are assigning the same headers 3 times on this file.